pe/orchestration.py

```diff
@@ -66,10 +66,10 @@ def get_sub_dicts_for_exam(self, page_size: int = 50) -> List[Dict[str, Any]]:
         get_subs_url: str = f'{CANVAS_URL_BEGIN}/courses/{self.exam.course_id}/students/submissions'
 
         canvas_params: Dict[str, Any] = {
-            'student_ids': ['all'],
-            'assignment_ids': [str(self.exam.assignment_id)],
+            'student_ids[]': 'all',
```

Comment:
Yeah, it is confusing, which is partly why I had it "wrong" before. If you look at issue #47, I link to a Canvas community post where someone asks this question, and the [answer seems to be that you just repeat the parameter name in the URL](https://community.canvaslms.com/t5/Question-Forum/How-do-I-construct-API-Command-URL-with-parameters-array/m-p/132695). That wouldn't work though with our Pythonic approach (I don't think) because dictionary keys have to be unique!